diff --git a/node_modules/react-voice-recorder/dist/index.css b/node_modules/react-voice-recorder/dist/index.css
index ed90905..0976032 100644
--- a/node_modules/react-voice-recorder/dist/index.css
+++ b/node_modules/react-voice-recorder/dist/index.css
@@ -25,7 +25,7 @@
   position: fixed;
   justify-content: center;
   align-items: center;
-  background: rgb(245, 0, 87);
+  background: #de2706;
   border-radius: 50%;
   bottom: 65px;
   right: 20%;
@@ -35,7 +35,7 @@
 ._1ceqH ._1lB9c{
     display: flex;
     justify-content: space-between;
-    background: #bd9f61;
+    background: #de2706;
     align-items: center;
     padding: 20px 20px;
     color: #fff;
@@ -84,23 +84,23 @@
   outline: 0;
   padding: 10px 20px;
   border-radius: 20px;
-  background: #185fec;
+  background: #de2706;
   color: #fff;
   cursor: pointer;
-  border:1px solid #185fec;
+  border:1px solid #de2706;
   transition:.3s ease all;
 }
 ._1ceqH ._1Yplu ._1Pz2d:hover{
   background: #fff;
-  color: #185fec;
+  color: #de2706;
 }
 ._1ceqH ._1Yplu ._2gd2_{
   background: #fff;
-  color: #185fec;
+  color: #de2706;
   margin-left:15px;
 }
 ._1ceqH ._1Yplu ._2gd2_:hover{
-  background: #185fec;
+  background: #de2706;
   color: #fff;
 }
 ._1ceqH ._f2DT8{
@@ -144,5 +144,5 @@
   background: #940505;
 }
 ._1ceqH ._qxztz ._3nQu5 {
-  background: #9c6702;
+  background: #de2706;
 }
\ No newline at end of file
diff --git a/node_modules/react-voice-recorder/dist/index.modern.js b/node_modules/react-voice-recorder/dist/index.modern.js
index 6e2e9bb..62a82cd 100644
--- a/node_modules/react-voice-recorder/dist/index.modern.js
+++ b/node_modules/react-voice-recorder/dist/index.modern.js
@@ -184,6 +184,8 @@ var Recorder = /*#__PURE__*/function (_Component) {
       chunks: this.chunks,
       duration: this.state.time
     });
+    //Added so that upload is triggered.
+    this.props.handleAudioUpload(blob)
   };
 
   _proto.render = function render() {
@@ -209,23 +211,11 @@ var Recorder = /*#__PURE__*/function (_Component) {
       className: styles.reco_header
     }, /*#__PURE__*/React.createElement("h2", {
       className: styles.h2
-    }, title), /*#__PURE__*/React.createElement("span", {
-      className: styles.close_icons
-    }, /*#__PURE__*/React.createElement("img", {
-      src: closeIcons,
-      width: 20,
-      height: 20,
-      alt: "Close icons"
-    }))), !medianotFound ? /*#__PURE__*/React.createElement("div", {
+    }, title)), !medianotFound ? /*#__PURE__*/React.createElement("div", {
       className: styles.record_section
     }, /*#__PURE__*/React.createElement("div", {
       className: styles.btn_wrapper
     }, /*#__PURE__*/React.createElement("button", {
-      onClick: function onClick() {
-        return _this4.props.handleAudioUpload(_this4.state.audioBlob);
-      },
-      className: styles.btn + " " + styles.upload_btn
-    }, "Upload"), /*#__PURE__*/React.createElement("button", {
       onClick: function onClick() {
         return _this4.handleRest();
       },
